zoo/utils.py

```diff
@@ -6,10 +6,9 @@
 
 def _get_app_version():
     """Return current commit SHA of the app."""
-    file_path = Path(__file__)
-    for directory in file_path.parents:
-        if (directory / ".git").exists():
-            return raven.fetch_git_sha(str(directory))
+    basedir = Path.cwd().parent
```

Comment:
Is this really a good fix? Or what was wrong with the previous fix? Because cwd can be completely different from where the `.git` repository is located...

Reply:
Fair enough, I assumed that we'd always run it from the root folder, but you are right, it might be a different case. Restored the `__path__` as root for the `.git` discovery

Reply:
The problem is that `.parents()` would recursively "escalate" up the ancestors, picking up an incorrect folder where to look into the version

Reply:
Yeah, I thought that is the issue, thank you for the explanation.